src/wren/StaticMesh.cpp

```diff
@@ -1933,6 +1933,9 @@ namespace wren {
 
         if (mCacheData->mGlNameBufferUnwrappedTexCoords)
           glDeleteBuffers(1, &mCacheData->mGlNameBufferUnwrappedTexCoords);
+
+        if (mCacheData->mGlNameBufferShadowCoords)
+          glDeleteBuffers(1, &mCacheData->mGlNameBufferShadowCoords);
```

Comment:
I believe the test is not needed as this buffer is systematically created (similar to `mGlNameBufferIndices`):
```suggestion
        glDeleteBuffers(1, &mCacheData->mGlNameBufferShadowCoords);
```